src/callbacks/amr_dg2d.jl

```diff
@@ -0,0 +1,330 @@
+
```

Comment:
That's something we have to discuss. This part is a specific combination of both an AMR callback and a DGSEM-based semidiscretization. From my point of view, it's equally justified to store it on the AMR side or the DGSEM side.

Reply:
The same is true for all other files `some_callback_dgNd.jl`.

Reply:
I guess many people will have rather strong opinions about this and it isn't possible to do it right so that everybody agrees. I just had to pick something at first.

Reply:
We opened an issue for this yesterday #236, so I think we can move the discussion there.